ui/app/styles/core/progress.scss

```diff
@@ -21,4 +21,5 @@
 // style the progress bar
 .progress[value]::-webkit-progress-value {
```

Comment:
Could we target `::-moz-progress-value` as well so there is cross-browser support? Looks like we may need  `-webkit-appearance: none;` on the .progress styles too. Ref: https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-progress-value

Reply:
ah yes, thank you so much for catching this! i often forget to add these additional prefixes. appreciate the link, too!